topo/node/srl/srl.go

```diff
@@ -134,10 +139,12 @@ func (n *Node) GenerateSelfSigned(ctx context.Context) error {
 func (n *Node) ConfigPush(ctx context.Context, r io.Reader) error {
 	log.Infof("%s - pushing config", n.Name())
 
-	cfg, err := io.ReadAll(r)
-	cfgs := string(cfg)
+	cfgBytes, err := io.ReadAll(r)
+	// replace quotes in the config with escaped quotes, so that we can echo this config
+	// via `echo` CLI commands.
+	cfg := strings.ReplaceAll(string(cfgBytes), `"`, `\"`)
 
-	log.V(1).Infof("config to push:\n%s", cfgs)
+	log.V(1).Infof("config to push:\n%s", cfg)
```

Comment:
Shouldn't these lines be after the check for err?

Reply:
thanks @bormanp 
done in [4cb5267](https://github.com/openconfig/kne/pull/343/commits/4cb5267965a65c22fc0c75dd3769ce06b3caeea5)